### DISCUSSION TITLE
How to batch replace the package?

### DISCUSSION THREAD

**Discussion by qiliangya** [verdaccio/verdaccio#3261] [Q&A] [answered]
I have a lot of private package, like @best/xxx

How to replace it is like @better/xxx

I tried to change the package name under "storage", but not work :(

**Comment by juanpicado** [comment, marked as answer]
The way to make changes is via the package manager, if you do things manually you might break things. But I understand the reason, by only modifying the package folder won't work, inside the folder, you still have `package.Json` files with metadata using the old name and the tarball also using the old name. Changing all of it depending on how much you have might be a pain. 

Let's imagine you did it, so the reaming change would be the hidden database name file, the old name would need to be replaced also there. I hope it helps but I recommend publishing them under a new name from scratch rather than all that manual work.
